cypress/page/shipping-step.page.ts

```diff
@@ -0,0 +1,19 @@
+class ShippingStepPage {
+    private checkbtn: string;
+    private termsCheck: string;
+  
+    constructor() {
+      this.checkbtn = ".cart_navigation > .button > span";
+      this.termsCheck = "#cgv";
+    }
+  
+    public check(): void {
+      cy.get(this.termsCheck).click();
+    }
+  
+    public pcheckout(): void {
+      cy.get(this.checkbtn).click();
+    }
+  }
+  
+  export {ShippingStepPage};
```

Comment:
EOF

Reply:
Also please check identation, I think the bracket that closes the class and the export have 1 extra identation

Reply:
done @danielgalvis98